editor/src/services/DataSourcesService/ConfigViews/ApiConfig.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <v-card class="px-1" color="transparent" elevation="0">
+    <v-row>
+      <v-simple-table>
+        <template v-slot:default>
+          <thead>
+            <tr>
+              <th class="text-left">Data input source</th>
+              <th class="text-left">Dest</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-for="item in dataInputVariables" :key="item.dest">
+              <td>
+                <v-combobox
+                  dense
+                  :items="items"
+                  class="caption"
+                  v-model="item.src"
+                  @change="getData"
+                  >{{ item.src }}
+
+                  <v-icon slot="append" color="accent darken-1">
+                    mdi-arrow-right-bold-box</v-icon
+                  >
+                </v-combobox>
+              </td>
+              <td>
+                <span class="caption">
+                  {{ item.dest }}
+                </span>
+              </td>
+            </tr>
+          </tbody>
+        </template>
+      </v-simple-table>
+    </v-row>
+    <v-row>
+      <v-col>
+        <v-text-field
+          v-model="data.value"
+          persistent-placeholder
+          :label="titles.id"
+          outlined
+          dense
+          class="caption"
+          @change="getData"
+        >
+        </v-text-field>
+      </v-col>
+      <v-col>
+        <v-text-field
+          v-model="data.label"
+          persistent-placeholder
+          dense
+          outlined
+          :label="titles.value"
+          class="caption"
+          @change="getData"
+        >
+        </v-text-field>
+      </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref, onMounted, inject } from '@vue/composition-api';
+const ApiConfig = defineComponent({
+  name: 'ApiConfig',
+  components: {},
+  props: ['dataConfig', 'dataSource'],
```

Comment:
What kind of data is `dataConfig` and `dataSource`?
If is possible, validate props like next way
`
props: {
  dataConfig: Object,
  dataSource: Array,
}
`